HW03/ivan_klimov/spec/hw_03_task_1_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+require_relative '../ivan_klimov_hw_03_t_01'
+
+RSpec.describe Homework3 do
+  subject { described_class.new }
```

Comment:
you can do this way and simplify your expects
```suggestion
  subject { described_class.new(log) }
```